lib/search_terms/keyword.rb

```diff
@@ -1,8 +1,14 @@
 class Keyword
-  def initialize(string)
+  def initialize(string,mode = :sensitive)
+    @mode = mode
     @string = string
   end
   def matches?(line)
-    line.include?(@string)
+    if @mode == :insensitive
```

Comment:
I don't like this if. Is there a better way? Should these be two classes? Is there a Ruby trick I don't (yet) know?